tests/_tests_dl_callbacks/config21.yml

```diff
@@ -0,0 +1,50 @@
+model_params:
```

Comment:
do we really need new config? could we just reuse some of the old ones?

Reply:
thanks,
nice point, probably we can update existing config - add KeyboardInterrupt callback and use env variable for interrupting.